kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/BaseConnectorIT.java

```diff
@@ -211,9 +215,10 @@ to listObjectsV2() returns a maximum of 1000 files.
    * @param expectedFiles the list of expected filenames for exact comparison
    * @return whether all the files in the bucket match the expected values
    */
-  protected boolean fileNamesValid(String bucketName, List<String> expectedFiles) {
+  protected void assertFileNamesValid(String bucketName, List<String> expectedFiles) {
     List<String> actualFiles = getBucketFileNames(bucketName);
-    return expectedFiles.equals(actualFiles);
+    ListAssert<String> listAssert = new ListAssert<>(actualFiles);
```

Comment:
Nit: no need to explicitly create the list assert, this could be something like `assertThat(actualFiles).containsExactly...`.

Reply:
done